site/docs/design_tokens/box-shadow.mdx

```diff
@@ -0,0 +1,39 @@
+---
+name: Box-shadow
+menu: Design tokens
+route: /design-tokens/box-shadow
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+# Box-shadow
+
+<LargeParagraph>Box-shadows are used to cast shadows from interface elements to backgrounds or other elements in order to allow visual cohesion and a more accessible experience.</LargeParagraph>
+
+Shadows under elements can be used to create visual distinction for content that otherwise would be hard to distinguish for the visually impaired or otherwise remain unnoticed.
+
+## Principles
+- **Shadows are not used to signify elevation in HDS.** Elements that require distinction can have shadows applied.
+- **Do not use shadows to highlight elements.** Elements with shadow around them should only have it for visual clarity.
+- **Never use shadows on typography (heading nor body).** Primary typographic contrast should always rely on colour contrast. 
+- **Never use shadows for effect.** For the City of Helsinki brand to remain intact, do not rely on style based effects of any kind.
+
+## Usage
+
+Shadows should be used very sparingly if at all and always only to allow greater visual contrast between elements. Do not use shadows as effects or as any other means of visual trickery.
```

Comment:
I would leave out the "or as any other means of visual trickery"

Reply:
Why? Do you mean the tone of voice is too playful for your taste, or some other reason?

Reply:
Omasta mielestäni ilmaus on vähän liian monitulkintainen, että mikä luetaan "trickeryksi" ja mikä ei. Lisäksi ei ehkä kieleltään vastaa muuta HDS:n dokumentaatiota.

Reply:
Yes, mostly the tone of voice is not in line with other guidelines we have.